docs/develop/unified-manifest-overview.md

```diff
@@ -94,7 +94,7 @@ The following tables lists which Office platforms directly support add-ins that
 | Office on mobile | Not directly supported |
 
 > [!NOTE]
-> If you're deploying an add-in in the [Microsoft 365 Admin Center](../publish/publish.md) and require it to run on platforms that don't directly support the unified manifest, the add-in must be a published AppSource add-in. Custom add-ins or line-of-business (LOB) add-ins that use the unified manifest can't currently be deployed in the Microsoft 365 Admin Center.
+> Custom add-ins or line-of-business (LOB) add-ins that use the unified manifest can't currently be deployed in the Microsoft 365 Admin Center.
```

Comment:
This might oversimplify a bit. An admin can obtain an add-in from AppSource and deploy it to all users from the Admin Center. This works even for add-ins that use the unified manifest. The hidden XML manifest that is generated is used on clients that don't support the JSON manifest.

Reply:
You're right. Reverted this change.